examples/google-cloud-speech/output/google-cloud-speech/google-cloud-speech.gemspec

```diff
@@ -0,0 +1,30 @@
+# -*- ruby -*-
+# encoding: utf-8
+
+Gem::Specification.new do |gem|
+  gem.name          = "google-cloud-speech"
+  gem.version       = "0.1.0"
+
+  gem.authors       = ["Google LLC"]
+  gem.email         = "googleapis-packages@google.com"
+  gem.description   = "google-cloud-speech is the official library for Cloud Speech API."
+  gem.summary       = "API Client library for Cloud Speech API"
+  gem.homepage      = "https://github.com/googleapis/googleapis"
+  gem.license       = "Apache-2.0"
+
+  gem.platform      = Gem::Platform::RUBY
+
+  gem.files         = `git ls-files -- lib/*`.split("\n") +
+                      ["README.md", "LICENSE", ".yardopts"]
+  gem.require_paths = ["lib"]
+
+  gem.required_ruby_version = ">= 2.0.0"
+
+  gem.add_dependency "google-gax", "~> 1.3"
+
+  gem.add_development_dependency "minitest", "~> 5.10"
+  gem.add_development_dependency "redcarpet", "~> 3.0"
+  gem.add_development_dependency "rubocop", "~> 0.60"
```

Comment:
I think we should pin to minor version for rubocop (e.g. `~> 0.60.0`) since new minor versions often introduce significant changes that cause previously passing tests to fail.